internal/params/protocol_params.go

```diff
@@ -6,7 +6,7 @@ const (
 	// MinGasLimit ...
 	MinGasLimit uint64 = 5000 // Minimum the gas limit may ever be.
 	// GenesisGasLimit ...
-	GenesisGasLimit uint64 = 4712388 // Gas limit of the Genesis block.
+	GenesisGasLimit uint64 = 10712388 // Gas limit of the Genesis block.
```

Comment:
if not reverted. Better add a separate TestGenesisGasLimit because the original gasLimit can be changed for the mainnet genesis.